client/src/main/java/application/view/ViewInitialiser.java

```diff
@@ -12,29 +12,50 @@ public ViewInitialiser() {
 
   }
 
+//  /** Public method used to initialise stages from
```

Comment:
Can you tidy this up or do you want to keep that?

Reply:
Fixed lad 👍